examples/http/hello-world/package.json

```diff
@@ -19,7 +19,7 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
-    "@roadwork/dapr-js-sdk": "file:../../../build",
+    "@roadwork/dapr-js-sdk": "../../../src",
```

Comment:
Why did you change this back to src?

Reply:
i want to test changes with running Dapr instance

Reply:
I added `npm run start:dev` to the root repository, normally that should help with testing code changes in the core. For examples the `file:../../../build` should work if you run `npm run build` on the root and `npm install` on the example, which will create a symlink to the build files so they stay up-to-date whenever you rebuild the core.